pkg/orchestrator/scanestimationwatcher/watcher.go

```diff
@@ -248,10 +293,16 @@ func (w *Watcher) reconcilePending(ctx context.Context, scanEstimation *models.S
 	}
 	logger.Debugf("%d Asset(s) have been created for Scan estimation", numOfAssets)
 
+	// Set default ttl if not set.
+	if scanEstimation.TTLSecondsAfterFinished == nil {
+		scanEstimation.TTLSecondsAfterFinished = utils.PointerTo(DefaultScanEstimationTTLSeconds)
+	}
+
 	scanEstimationPatch := &models.ScanEstimation{
-		StartTime: utils.PointerTo(time.Now()),
-		AssetIDs:  scanEstimation.AssetIDs,
-		State:     scanEstimation.State,
+		StartTime:               utils.PointerTo(time.Now()),
```

Comment:
you are missing the `endTime` and the `DeleteAfter` here?

Reply:
No, This is only pending reconcile, the endTime and deleteAfter are set when the scan is done (see reconcileInProgress)